.github/workflows/autopr_merged.yaml

```diff
@@ -0,0 +1,90 @@
+name: "Tag and release to production"
+
+on: 
+  pull_request:
+    types: [closed]
```

Comment:
Interesting change of the flow. I was expecting we would rather have the VERSION merge be the latest manual step but yeah, we can keep it the way it is right now with the manual merge of the pr-bot changes and then automate the rest.

Reply:
i figured it's more intuitive when someone merges the PR bot release that a release would happen and not have to worry about tagging and updating version files.